aries-site/src/examples/cardPreviews/toast-notification.js

```diff
@@ -36,3 +37,9 @@ export const ToastPreview = ({card, message, title}) => (
     </Box>
   </Box>
 );
+
+ToastPreview.propTypes = {
+  card: PropTypes.string,
```

Comment:
```suggestion
  card: PropTypes.boolean,
```
was using this as a boolean flag